### DISCUSSION TITLE
Best way to Organize multiple Client Installs?

### DISCUSSION THREAD

**Discussion by eduncan911** [netbox-community/netbox-docker#466] [General]
New to NetBox, I'm starting to notice that all Sites are intermixed throughout the other organization features.  It seems Netbox is missing a higher-order "Customer" concept?  Doesn't django support multiple "sites"?  Anyhow, that's a question for Netbox...

For `netbox-docker`, I am wondering if there's an option to specify different docker installs - per customer - within a single installation.  Right now, if I continue down the path I am going, I'll have a few dozen `-p` project installs of Docker images - which is quite excessive.

```
$ docker-compose -p myclient1 up
```

Is there a way to run just a single instance of NetBox?

Or, should I continue to special all "customers" as "sites", even though several customers has *multiple sites*?

**Comment by eduncan911** [comment]
Ah, that's what Tenants are for!

MODS: Feel free to delete this discussion.

**Reply by tobiasge** [reply]
You can separate the information with tenants. But you have to be very careful with the access right.
If you ever plan to give your customers access to Netbox you should consider separate installations per customer.

**Reply by eduncan911** [reply]
I'm hoping I can export/produce "snapshots" of the description of the network.  As customers should not be touching the networks, or at least if they do they have a printed reference.
